packages/workspace/src/browser/quick-open-workspace.ts

```diff
@@ -75,12 +82,11 @@ export class WorkspaceQuickOpenItem extends QuickOpenItem {
     }
 
     /**
-     * Display the workspace path as part of
-     * the WorkspaceQuickOpenItem description
+     * Tildify workspace path for display purposes
      * @returns workspace path
      */
     getDescription(): string {
-        return (this.workspace) ? new URI(this.workspace).path.dir.toString() : '';
+        return (this.home) ? WorkspaceUtils.tildifyPath(this.workspace, this.home) : new URI(this.workspace).path.toString();
```

Comment:
Just wondering, is it actually useful to have

    new URI(this.workspace).path.toString()

vs just

    this.workspace

?

Reply:
the only reason I return `new URI(this.workspace).path.toString()` instead of `this.workspace` is that `this.workspace` returns the entire path including `file:///....` which I didn't want to necessarily display to end users.